.travis.yml

```diff
@@ -2,6 +2,9 @@ language: erlang
 
 otp_release:
   - 18.0
+  - 18.1
+  - 18.2.1
```

Comment:
There are bugfixes in all the patch versions. Why not run `18.2.4.1` or `18.0.5`?